applications/asset_tracker_v2/doc/cloud_wrapper.rst

```diff
@@ -0,0 +1,162 @@
+
+.. note::
+   The *Direction* entry in table :ref:`Data routing tables <data_filtering>` refers to the direction of the data flow.
+   C2D stands for *Cloud-to-Device* and D2C stands for *Device-to-Cloud*.
+
+.. _data_filtering:
+
+AWS IoT topics
+==============
+
++------------------------------+----------------------------------------------------+-----------+
+|              Data            |            Topic                                   | Direction |
++==============================+====================================================+===========+
+| A-GPS requests               | <imei>/agps/get                                    | D2C       |
++------------------------------+----------------------------------------------------+-----------+
+| P-GPS requests               | <imei>/pgps/get                                    | D2C       |
++------------------------------+----------------------------------------------------+-----------+
+| Neighbor cell measurements   | <imei>/ncellmeas                                   | D2C       |
++------------------------------+----------------------------------------------------+-----------+
+| Button presses               | <imei>/messages                                    | D2C       |
++------------------------------+----------------------------------------------------+-----------+
+| Sensor/device data           | $aws/things/<imei>/shadow/update                   | D2C       |
++------------------------------+----------------------------------------------------+-----------+
+| Device configuration         | $aws/things/<imei>/shadow/update                   | D2C       |
++------------------------------+----------------------------------------------------+-----------+
+| Buffered sensor/device data  | <imei>/batch                                       | D2C       |
++------------------------------+----------------------------------------------------+-----------+
+| A-GPS response               | <imei>/agps                                        | C2D       |
++------------------------------+----------------------------------------------------+-----------+
+| P-GPS response               | <imei>/pgps                                        | C2D       |
++------------------------------+----------------------------------------------------+-----------+
+| Device configuration updates | $aws/things/<imei>/shadow/delta                    | C2D       |
+|                              +----------------------------------------------------+           |
+|                              | $aws/things/<imei>/shadow/get/accepted             |           |
+|                              +----------------------------------------------------+           |
+|                              | $aws/things/<imei>/shadow/get/accepted/desired/cfg |           |
++------------------------------+----------------------------------------------------+-----------+
```

Comment:
**suggestion**: to improve readability, split table in two, one for D2C and one for C2D, remove Direction column.

Reply:
Maybe also have the topics between double backticks. In all tables.